api/class-wc-calypso-bridge-settings-email-groups-controller.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * REST API Settings controller
```

Comment:
Maybe this description could reflect that it is for email settings specifically?